Dockerfile

```diff
@@ -10,6 +10,9 @@ ARG RUNTIME_PACKAGES
 ARG RUNTIME_TAR_PACKAGES
 ARG RUNTIME_PIP_PACKAGES
 
+COPY --from=build-image /usr/local/bin/lnls-single-uid /usr/local/bin/lnls-single-uid
+RUN lnls-single-uid
+
```

Comment:
I'm not sure I agree with this being a `COPY`. (including for the base image)

It's such a short command...

I'd be okay with this copying the rule from the base image, though.